schemas/com.snowplowanalytics.mobile/application_install/jsonschema/1-0-0

```diff
@@ -0,0 +1,14 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for an event where a mobile application is installed.",
+	"self": {
+		"vendor": "com.snowplowanalytics.mobile",
+		"name": "application_install",
+		"format": "jsonschema",
+		"version": "1-0-0"
+	},
+	"type": "object",
+	"properties": {
+	},
```

Comment:
just a q, what's the point of a schema without properties?

Reply:
Assuming its `Map[String, Any]`. Technically, `patternProperties` could express it better, but having few troubles with `patternProperties` in Iglu - I now tend to advise against it.